src/components/StoryCreationForm/StoryCreationForm.tsx

```diff
@@ -0,0 +1,66 @@
+"use client"
+import { useForm } from "react-hook-form"
+import { Form, FormField, FormItem, FormLabel, FormControl } from "../ui/form"
+import { Input } from "../ui/input"
+import { Textarea } from "../ui/textarea"
+import { Button } from "../ui/button"
+import type { StoryData } from "@/app/create/page"
+
+const StoryCreationForm = ({
+  onPostAction
+}: {
+  onPostAction: (data: StoryData) => Promise<void>
+}) => {
+  const onSubmitHandler = (data: StoryData) => {
+    onPostAction(data)
+  }
+  const form = useForm({
+    defaultValues: {
+      title: '',
+      openingSegment: ''
+    }
+  })
+  return <Form {...form} >
+    <h1 className='text-lg lg:text-4xl font-bold mb-12'> Create a story </h1>
```

Comment:
```suggestion
<h1 className='text-lg lg:text-[1.75rem] font-bold mb-12'> Create a story </h1>
```
`lg:text-4xl` does not match fontsize of 28px(i.e 1.75rem)  at large screen (from the figma design). Instead it should be  `lg:text-[1.75rem]`.

Putting things in `[]` is how you provide custom one-off values in tailwindcss